src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -5,4 +5,25 @@ class SimidMapCreative extends BaseSimidCreative {
   constructor() {
     super();
   }
+  /** @override */
+  onStart(eventData){
+    super.onStart(eventData);
+    this.adParams = JSON.parse(this.creativeData.adParameters);
+    console.log("Ad params: ", this.adParams);
+    var paramObject = this.adParams[0];
+    console.log(paramObject);
+    var buttonParam = paramObject["buttonParam"];
+    console.log(buttonParam);
+    this.specifyButtonText(buttonParam);
+  }
+ 
+  specifyButtonText(buttonParam) {
+    var findNearestButton = document.getElementById('findNearest');
+    findNearestButton.innerText = "Find Nearest " + buttonParam;
+    findNearest.onclick = () => displayMap();
+  }
+ 
+  displayMap() {
+  //To Do -- Kristen implement this 
```

Comment:
use the form

`// TODO(kristenmason@): ...`
or even better, if you create a github issue for it then you can do
`// TODO(https://linkToIssue): ...`

Reply:
did the first way, need to look into how to link issues on github